glide.yaml

```diff
@@ -93,7 +93,11 @@ import:
   version: ded73eae5db7e7a0ef6f55aace87a2873c5d2b74
 # master
 - package: github.com/coreos/etcd
-  version: v3.2.8
+  version: v3.2.16
+- package: google.golang.org/grpc
```

Comment:
this is pinning the version of grpc to the version used by etcd 3.2.16

Reply:
Shouldn't that be discovered by glide?

Reply:
surprisingly not... when I just bumped etcd, these were unchanged and also etcd/vendor dir was unchanged which is pretty weird.

Reply:
What is strange is that the HEAD version of grpc is 1.3.0, but the etcd 3.2.8 version of grpc is 1.2.1 (from https://github.com/coreos/etcd/blob/v3.2.8/glide.yaml#L99). That means that in HEAD we are ahead of what our level of etcd was using.

Reply:
> when I just bumped etcd, these were unchanged (on disk).

did you unpin these first? if not, pinned means pinned.

Reply:
@liggitt i don't see how they are currently pinned, we don't mention them in glide.yaml